apps/microsoft-teams/frontend/src/components/config/NotificationsSection/NotificationsSection.tsx

```diff
@@ -127,36 +126,36 @@ const NotificationsSection = (props: Props) => {
         />
       </Box>
       <ContentTypeContextProvider>
-        {notifications.map((notification, index) => {
-          const inEditMode = notificationIndexToEdit === index;
+        <ChannelContextProvider>
```

Comment:
It might be worth discussing lifting these providers into a `Providers` component and wrapping higher in component tree 🤔 just a call out since we have multiple providers now, good code huddle topic maybe

Reply:
Agreed! I'll keep it as is for now, and we can discuss an approach later. We can clean up a bit when we address some additional error handling later